### DISCUSSION TITLE
Can RASPAP and Samba coexist ?

### DISCUSSION THREAD

**Discussion by ckl67** [RaspAP/raspap-webgui#1644] [Q&A] [answered]
Raspberry Pi 3 Model B
Rasberry Pi OS Lite 64 bits
RASPAP
	curl -sL https://install.raspap.com | bash

Hello,

Could you tell me if there's a solution for combining RASPAP with a file server? 
I've tried Samba and NFS, but nothing seems to work. 
The aim is to be able to connect to my Rasberry over WiFi (without my Rasperry necessarily being connected to the LAN) and to be able to transfer files to it.

The ideal solution would be Samba, but I haven't found out whether it's possible for RASPAP and Samba to coexist.

Thanks for your help and advice 
Regards Christian

**Comment by billz** [comment, marked as answer]
Samba works perfectly well with RaspAP - you'll find many examples of offline file servers built this way. After installing them, connect to RaspAP's WiFi network then access your Samba share with `smb://10.3.141.1/share` (Linux) or `\\10.3.141.1\share` (Windows).

As one example, this project uses IPFS (instead of Samba) together with RaspAP https://likebutter.app/box/

**Comment by ckl67** [comment]
Works perfectly ..
Thank you.
The issue was due to https://github.com/RaspAP/raspap-webgui/discussions/1643

With an Ubuntu PC it is working as a charm !
What a time I lost ... :-(

Merci

**Comment by ckl67** [comment]
without pretension, a simple cohabitation project between RASPAP and Samba
https://github.com/ckl67/lapem

**Reply by billz** [reply]
génial!
